config/log4j-rolling.properties

```diff
@@ -65,3 +65,6 @@ log4j.logger.processing=ERROR, kafka_appender
 log4j.additivity.processing=false
 
 log4j.logger.org.reflections=ERROR, main
+
+# To acheive high throughput on pull queries, avoid logging every request from Jetty
+log4j.logger.io.confluent.rest-utils.requests=WARN
```

Comment:
Is this the config that production deployments use as well? Wondering if its good to document this under some troubleshooting docs?  https://docs.ksqldb.io/en/latest/operate-and-deploy/#troubleshooting ? 

Don't know if it makes sense. Leave it you to make the final call.

Reply:
When you say production deployments, do you mean CCloud?

Reply:
No.. the docker image or however external ksqlDB users configure the server

Reply:
Will make a change here as well: https://github.com/confluentinc/ksql-images/blob/master/cp-ksql-server/include/etc/confluent/docker/log4j.properties.template